docs/platform/howto/byoc/networking-security.md

```diff
@@ -0,0 +1,123 @@
+| Bastion subnet | Aiven Management | Communication channel with the Aiven Management plane used to collect metrics and logs from the bastion and workload nodes              |
+| Bastion subnet | Aiven Management | Required for the bastion and workload nodes to make calls to the Aiven Management plane                                                 |
+| Bastion subnet | AWS CloudFront   | Communication channel for downloading RPM packages for bastion and workload nodes setup                                                 |
+| Bastion subnet | Workload subnet  | Aiven uses SSH from the bastion subnet to the workload subnet for troubleshooting, setup, or configuration tasks on the workload nodes. |
+| Bastion subnet | Workload subnet  | Aiven Management plane accesses the workload nodes via the bastion node to retrieve status information and perform routine operations.  |
+| Bastion subnet | DNS & NTP        | Destination dependant on cloud provider                                                                                                 |
+
+:::note
+Aiven Management IPs are considered relatively stable but do occasionally change and are
+considered dynamic for the purposes of firewall policies. To that end, an egress of
+`0.0.0.0/0` is generally required to accommodate traffic to Aiven Management.
+:::
+
+## Workload nodes networking
+
+Management traffic to and from workload nodes is sent through the bastion proxy and
+encrypted. Aiven Management and API calls are always encrypted. SSH access is permitted if
+Aiven support staff require direct access to the bastion or workload nodes. SSH
+connections from support staff are logged, monitored, and require validation by Aiven’s
+security operations team. SSH connections originate from Aiven Management plane's NAT gateways.
+
+### Traffic to the workload
+
+On top of SSH and Aiven Management Plane connectivity, there are also ingress and egress
+rules from the Customer Networks that need to be defined to allow customer applications to
+connect to Aiven services. This depends on the cloud provider being used, and the
+customer’s network architecture.
+
+| Source                   | Destination     | Description                                                                                                                             |
+| ------------------------ | ----------------| --------------------------------------------------------------------------------------------------------------------------------------- |
+| Bastion subnet           | Workload subnet | Aiven uses SSH from the bastion subnet to the workload subnet for troubleshooting, setup, or configuration tasks on the workload nodes. |
+| Bastion subnet           | Workload subnet | Aiven Management plane accesses the workload nodes via the bastion node to retrieve status information and perform routine operations.  |
```

Comment:
These could be grouped together and just refer back to the description of this traffic in the "Traffic from the bastion" section, I think. The same thing is described in two places.

Reply:
I assume not everyone is going to read throughout the whole document and some people might be only interested in a single section. We'd like to avoid making them jump back and forth to look somewhere else for info that's why including it also here for ease of accessibility.
About the grouping, again I defer to @jeremy-heller-aiven & @kevinmfox ...